modules/EnsEMBL/Web/Object/Variation.pm

```diff
@@ -87,15 +87,28 @@ sub get_external_links {
 
   my $attribs = $self->Obj->get_all_attributes();
 
-  if ($attribs->{'cerealsdb_external_links'}){
-
-
-    my $link = qq{<a href="$attribs->{'cerealsdb_external_links'}">Additional details from CerealsDB</a>};
-    $self->{'external_link'} = "$link</ul>";
+  ##Get necessary attribs for external links
+  my $html_out;
+  my $external_link = $attribs->{'cerealsdb_external_links'};
+  my $qtl_link      = $attribs->{'cerealsdb_qtl'};
+
+  ##Removing extra comma at the end of the link if it appears
+  if ($external_link){
+        $external_link =~ s/,$//g
+  }
 
+  ##Just external link to CerealsDB
+  if ($external_link){
+    $html_out = qq{<a href="$external_link">Additional details from CerealsDB</a>};
   }
 
+  ##QTL link as well
+  if ($qtl_link){
```

Comment:
Hi @gnaamati, could you confirm if `$external_link` will always be defined when `$qtl_link ` is defined, please?

Reply:
Yes, these are both links to cerealDB, and external links are always defined in this case.
In the future if we need to link out to other sources we will make this code more generic.